openid-sharedsignals-framework-1_0.md

```diff
@@ -940,7 +940,7 @@ The HTTP POST request MAY contain the Receiver-Supplied values of the Stream
 Configuration ({{stream-config}}) object:
 
 * `events_requested`
-* `delivery` : Note that in the case of the POLL method, the `endpoint_url` value is
+* `delivery` : Note that in the case of the poll method, the `endpoint_url` value is
```

Comment:
I'm good with this change, but should we refer to the spec {{RFC8936}} here?

Reply:
felt a little repetitive with https://github.com/openid/sharedsignals/pull/169/files#diff-193e4a2db2f645f1b4f19c4a9f8facbef29f86e122a8da4c529542750a291a1cR892-R896 but happy to link it